src/components/Modals/FiatRamps/components/FiatRampActionButtons.tsx

```diff
@@ -1,48 +1,36 @@
-import { Button, ButtonGroup } from '@chakra-ui/react'
+import { Tab, TabList, Tabs } from '@chakra-ui/react'
+import { useMemo } from 'react'
 import { useTranslate } from 'react-polyglot'
 
 import { FiatRampAction } from '../FiatRampsCommon'
 
 export const FiatRampActionButtons = ({
   action,
   setAction,
-  supportsBuy,
-  supportsSell,
 }: {
   action: FiatRampAction
   setAction: (action: FiatRampAction) => void
-  supportsBuy: boolean
-  supportsSell: boolean
 }) => {
   const translate = useTranslate()
 
-  const commonButtonProps = {
-    pt: 4,
-    pb: 4,
-    pl: 10,
-    pr: 10,
-  }
+  const activeIndex = useMemo(() => {
+    if (action === FiatRampAction.Sell) {
+      return 1
+    } else {
+      return 0
+    }
+  }, [action])
```

Comment:
```suggestion
  const activeIndex = useMemo(() => Number(Boolean(action === FiatRampAction.Sell)), [action])
```

do you like golf beard?